mesheryctl/pkg/utils/helpers.go

```diff
@@ -237,13 +237,14 @@ func PreReqCheck(subcommand string) error {
 }
 
 func startdockerdaemon(subcommand string) error {
+	userResponse := false
 	// read user input on whether to start Docker daemon or not.
-	var userinput string
-	fmt.Printf("Start Docker now [y/n]? ")
-	fmt.Scan(&userinput)
-	userinput = strings.TrimSpace(userinput)
-	userinput = strings.ToLower(userinput)
-	if userinput == "n" || userinput == "no" {
+	if SilentFlag {
+		userResponse = true
+	} else {
+		userResponse = AskForConfirmation("Start Docker now?")
```

Comment:
Change "Start Docker now?" to "Start Docker now". i.e remove the question mark, since `AskForConfirmation` method already has a question mark.

Great spot with this error. And great work changing to the `AskForConfirmation` function.

Other than removing the question mark, rest LGTM. @hexxdump please make that tiny change and you can merge.

Reply:
> Change "Start Docker now?" to "Start Docker now". i.e remove the question mark, since `AskForConfirmation` method already has a question mark.
> 
> Great spot with this error. And great work changing to the `AskForConfirmation` function.
> 
> Other than removing the question mark, rest LGTM. @hexxdump please make that tiny change and you can merge.

Done.